package.json

```diff
@@ -43,10 +43,17 @@
     "@types/mocha": "^5.2.0",
     "@types/node": "^11.13.7",
     "@types/sinon": "^7.0.11",
+    "@typescript-eslint/eslint-plugin": "^3.7.1",
+    "@typescript-eslint/parser": "^3.7.1",
     "chai": "^4.1.2",
     "codecov": "^3.6.5",
     "danger": "^7.1.3",
+    "danger-plugin-eslint": "^0.1.0",
     "danger-plugin-tslint": "^2.0.0",
```

Comment:
Do we still need this?

Reply:
Yes, because the other packages still use `tslint`. We can remove after we are done.